CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * [#429](https://github.com/clojure-emacs/clojure-mode/issues/429): Fix a bug causing last occurrence of expression sometimes is not replaced when using `move-to-let`.
 * [#423](https://github.com/clojure-emacs/clojure-mode/issues/423): Make `clojure-match-next-def` more robust against zero-arity def-like forms.
 * [#451](https://github.com/clojure-emacs/clojure-mode/issues/451): Make project root directory calculation customized by `clojure-project-root-function`.
+* Numerous font-locking fixes and improvements
```

Comment:
List here the biggest user-visible changes explicitly and I guess we can merge this.